src/components/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import "./LandingPage.scss";
+import LoginButton from "../Login/LoginButton";
+import Button from "../Button/Button";
+import startIcon from "../../assets/start_icon.svg";
+import { FileIconHtml, FileIconPython } from "../../Icons";
+
+const LandingPage = () => {
+  const { t, i18n } = useTranslation();
+  const locale = i18n.language;
+  return (
+    <div className="landing-page-wrapper">
+      <div className="landing-page__projects">
+        <h1 className="landing-page__projects--title">
+          {t("landingPage.title")}
+        </h1>
+        <h2 className="landing-page__projects--subtitle">
+          {t("landingPage.subtitle")}
+        </h2>
+        <div className="landing-page__projects--buttons">
+          <Button
+            className="btn--primary landing-page__button"
+            buttonHref={`${locale}/projects/blank-python-starter`}
```

Comment:
`/` needed otherwise it adds to the current path, starting with a `/` forces it to base the path from the root
```suggestion
            buttonHref={`/${locale}/projects/blank-python-starter`}
```